js/piwik.js

```diff
@@ -2484,6 +2490,66 @@ if (typeof Piwik !== 'object') {
                 }
             }
 
+            /*
+             * Sets up the heart beat timeout.
+             */
+            function heartBeatUp(delay) {
+                if (heartBeatTimeout) {
```

Comment:
The check is only done in one place, `enableHeartBeatTimer()`. I want to keep it there for code clarity, since the heart beat should only be started after the first request is sent. If I remove it, it will seem like the heart beat is always started immediately after enableHeartBeatTimer() is called.